tests/backend/test_Piece.py

```diff
@@ -0,0 +1,59 @@
+from backend.Piece import *
+import pytest
+
+
+@pytest.fixture
+def piece():
+    piece = Piece(Turn.WHITE)
+    yield piece
+
+
+@pytest.fixture
+def pieces():
+    pieces = Pieces(9)
+    yield pieces
+
+class TestPiece:
+    def test_piece_initialization(self, piece):
+        # Test if the piece is initialized with the expected values
+        assert piece.get_turn() == Turn.WHITE
+        assert piece.get_position() == (-1, -1)
+
+    def test_piece_set_turn(self, piece):
+        piece.set_turn(Turn.BLACK)
```

Comment:
From a business logic perspective this should be `set_color`

Reply:
agreed